config/software/stunnel.rb

```diff
@@ -54,16 +54,21 @@
     env["WIN32_SSL_DIR_PATCHED"] = "#{install_dir}/embedded"
 
     if windows_arch_i386?
```

Comment:
Normally I would use `ENV["MSYSTEM"]` to determine whether to use `mingw32` or `mingw64`. The problem in this instance is that the make commands are `mingw` or `mingw64`. Those targets are specific to this software definition so I don't think we need to expose any helper in Omnibus (we already have `ENV["MSYSTEM"]`).

I'll cleanup the code now